networkx/algorithms/operators/all.py

```diff
@@ -50,6 +50,16 @@ def union_all(graphs, rename=()):
     If a graph attribute is present in multiple graphs, then the value
     from the last graph in the list with that attribute is used.
 
+    Examples
+    --------
+    >>> G1 = nx.Graph([(1, 2), (2, 3)])
+    >>> G2 = nx.Graph([(4, 5), (5, 6)])
+    >>> result_graph = union_all([G1, G2])
+    >>> result_graph.nodes()
+    [1, 2, 3, 4, 5, 6]
+    >>> result_graph.edges()
+    EdgeView([(1, 2), (2, 3), (4, 5), (5, 6)])
```

Comment:
```suggestion
    >>> G1 = nx.Graph([(1, 2), (2, 3)])
    >>> G2 = nx.Graph([(4, 5), (5, 6)])
    >>> result_graph = nx.union_all([G1, G2])
    >>> result_graph.nodes()
    [1, 2, 3, 4, 5, 6]
    >>> result_graph.edges()
    EdgeView([(1, 2), (2, 3), (4, 5), (5, 6)])
```

tiny nit pick about using `nx.union_all` instead of just `union_all` as the expectation is for users to be able to quickly copy this example and run it. If it's just `union_all` they need to explicitly import `union_all` first :)